src/main/resources/tree.json

```
@@ -78,9 +73,101 @@
     },
     {
       "key": "EXCEPT_M1",
-      "children": []
+      "children": [
+        {
+          "key": "N1_N3",
+          "children": [
+            {
+              "key": "N1_N3_WEIGHT",
+              "children": [
+                {
+                  "key": "AGE",
+                  "children": []
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "key": "M2_M3",
+          "children": [
+            {
+              "key": "ELECTRIC_TYPE_ENGINE",
+              "children": [
+                {
+                  "key": "AGE",
+                  "children": []
+                }
+              ]
+            },
+            {
+              "key": "GASOLINE_TYPE_ENGINE",
+              "children": [
+                {
+                  "key": "M2_M3_GASOLINE_ENGINE_VOLUME",
+                  "children": [
+                    {
+                      "key": "AGE",
+                      "children": []
+                    }
+                  ]
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "key": "TRUCK_UNITS",
+          "children": [
+            {
+              "key": "TRUCK_UNITS_EXCEPT_6_CLASS",
```

If I understand correctly what you would like to achieve, It's better to use for this cases `_OTHER`
```suggestion
              "key": "TRUCK_UNITS_OTHER",
```

---

In this case yes, we can rename - we have only 2 types of trucks: 6 class and not-6-class.
Renamed